src/braket/_schemas/_version.py

```diff
@@ -15,4 +15,4 @@
    Version number (major.minor.patch[-label])
 """
 
-__version__ = "1.7.3.dev0"
+__version__ = "1.7.2"
```

Comment:
These are handled by the release pipeline. Why is this being modified manually ?

Reply:
Changing to the last release in pypi

Reply:
Rolled back the changed to 1.7.3.dev0